lib/active_merchant/billing/gateways/card_connect.rb

```diff
@@ -0,0 +1,283 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class CardConnectGateway < Gateway
+      self.test_url = 'https://fts.cardconnect.com:6443/cardconnect/rest/'
+      self.live_url = 'https://fts.cardconnect.com:8443/cardconnect/rest/'
```

Comment:
@nfarve - The 8443 port *might* be an issue for us since it's non-standard. Just to be safe, it's probably worth checking with the ops team on this.

Reply:
Ticket up to whitelist both ports.

Reply:
We only need 8443 since that's the production endpoint. The test endpoint doesn't need to be whitelisted.

Reply:
Updated the ticket. Thanks for the catch.